tutorials/hist/NormalizeHistogram.C

```diff
@@ -0,0 +1,73 @@
+/// \file
+/// \ingroup tutorial_hist
+/// \notebook -js
+/// Normalizing a Histogram
+///
+/// Image produced by `.x Normalizing_Histogram.C`
+/// 2 different methods of normalizing Histograms
+/// are shown, each with the original Histogram
+/// next to the normalized one.
+/// \macro_image
+/// \macro_code
+///
+/// The methods are:
+/// 1:
+/// ```
+/// hist2->Scale(1. / hist2->Integral());
+///```
+/// 2:
+/// ```
+///  hist3->Scale(1. / hist3->Integral("width"));
+/// ```
+///
+/// \author Advait Dhingra
+
+#include "TH2F.h"
+#include "TRandom.h"
+#include "TCanvas.h"
+
+void NormalizeHistogram()
+{
+
+   // all of the histograms
+
+   TH1F *hist1 = new TH1F("Unnormalized Histogram", "Unnormalized Hist", 100, 0, 100);
+   hist1->SetStats(0);
+   TH1F *hist2 = new TH1F("Method 1 Normalized", "Method 1 Normalized", 100, 0, 100);
+   hist2->SetStats(0);
+   TH1F *hist3 = new TH1F("Method 2 Normalized", "Method 2 Normalized", 100, 0, 100);
```

Comment:
Please use more descriptive / telling / educational names and titles, esp in tutorials. Suggestions:

```suggestion
   TH1F *orig = new TH1F("orig", "Original histogram before normalization", 100, 0, 100);
   orig->SetStats(0);
   TH1F *norm = new TH1F("norm", "Histogram normalized with bin content", 100, 0, 100);
   norm->SetStats(0);
   TH1F *normw = new TH1F("normw", "Histogram normalized with bin content times bin width", 100, 0, 100);
```

Reply:
Yeah, thats better.